packages/dynamic-links/e2e/dynamicLinks.e2e.js

```diff
@@ -155,6 +155,7 @@ describe('dynamicLinks()', function () {
 
       dynamicLink.should.be.an.Object();
       dynamicLink.url.should.equal('https://rnfirebase.io');
+      dynamicLink.utmParameters.should.eql({});
```

Comment:
With a quick addition of utm params to TEST_LINK the population may be verified I think?